compiler/ras/Debug.hpp

```diff
@@ -594,6 +594,9 @@ class TR_Debug
    virtual void printRegisterDependencies(TR::FILE *pOutFile, TR_S390RegisterDependencyGroup *rgd, int numberOfRegisters);
    const char * getName(TR::RealRegister *, TR_RegisterSizes size = TR_WordReg);
 #endif
+#if defined(TR_TARGET_ARM64)
+   void print(TR::FILE *pOutFile, TR::UnresolvedDataSnippet * snippet);
+#endif
```

Comment:
Why is this needed, and where is it implemented?

Reply:
without this it does not compile. We could have the proper file stub or just have a temporary definition there. chose the later